typescript/sdk/src/deploy/router/AbacusRouterChecker.ts

```diff
@@ -11,9 +11,9 @@ import { RouterConfig } from './types';
 
 export class AbacusRouterChecker<
   Chain extends ChainName,
-  Contracts extends RouterContracts,
   App extends AbacusApp<Contracts, Chain>,
   Config extends RouterConfig,
+  Contracts extends RouterContracts,
```

Comment:
how does this build? `Contracts` is used by `App`

Reply:
Dunno but it builds fine and allows us to be consistent with the `AbacusAppChecker` order